1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export function palindromeChecker(phrase: string): boolean {
+    const noSpacePhrase = phrase.toLowerCase().replace(/\s+/g, '');
+    const charArray = noSpacePhrase.split('');
+
+    return charArray.reverse().join("") === noSpacePhrase;
+}
```

Comment:
I think the variable names could be improved here, maybe something like this would read better?

```typescript
const lowerCaseWithNoSpacesPhrase = phrase.toLowerCase().replace(/\s+/g, '');

const  reversedPhrase = lowerCaseWithNoSpacesPhrase.split('').reverse().join('')

return lowerCaseWithNoSpacesPhrase === reversedPhrase
```